bcgov/PIMS
# PIMS-2130: Update Python Create Jira Tickets Script

## 🎯 Summary

  
[PIMS-2130](https://citz-imb.atlassian.net/browse/PIMS-2130)


Removing script and portion of workflow related to the create jira ticket script. This process has been transferred to a webhook (sends information on any new issue to this repo) and Jira Automate. See the created automate process [here](https://citz-imb.atlassian.net/jira/software/c/projects/PIMS/settings/automate#/rule/22612522).


## Testing

1. Run the github action [`NPM Dependency Reports`](https://github.com/bcgov/PIMS/actions/workflows/app-npm-dep-check.yml) on this branch.
    a. Note that you will have to select this branch (PIMS-213--fix-ticket-creator-script) when manually running the action to ensure the ticket creator script doesn't run. 
2. Check the new tickets that are created. Easiest to check the tickets under [PIMS-450](https://citz-imb.atlassian.net/browse/PIMS-450) the NPM Dependency Update Epic.

## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
